src/api/controllers/rateArticleController.js

```diff
@@ -0,0 +1,49 @@
+import model from '../models';
+import statuses from '../../helpers/constants/status.codes';
+import sendError from '../../helpers/error.sender';
+
+const { Ratings } = model;
+
+/**
+ *
+ *
+ * @class RatingsController
+ */
+class RatingsController {
+  /**
+   *this is the controller which enables the user to rate a certain article
+   *
+   * @author Frank Mutabazi
+   * @static
+   * @param {object} req - this is the request object
+   * @param {object} res - this is the response object
+   * @returns {object} - this function returns an object
+   * @memberof RatingsController
+   */
+  static async rateArticle(req, res) {
+    const { slug } = req.params;
+    const { Rating } = req.body;
+    const { user } = req.user;
+
+    const { id } = user;
+
+    try {
+      await Ratings.create({
+        userId: id,
+        articleSlug: slug,
+        rating: parseInt(Rating, 10)
+      });
+
+      return res.status(statuses.OK).json({
+        status: statuses.OK,
+        message: `you rated ${slug} with ${Rating} rates, Thanks`
+      });
+    } catch (err) {
```

Comment:
what is the limit or max value of rating?

Reply:
so far I didn't think about it? what's your suggestion @alainburindi